src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/HotReload/ProjectHotReloadSessionManager.cs

```diff
@@ -14,15 +14,15 @@ namespace Microsoft.VisualStudio.ProjectSystem.HotReload;
 [Export(typeof(IProjectHotReloadUpdateApplier))]
 internal sealed class ProjectHotReloadSessionManager : OnceInitializedOnceDisposedAsync, IProjectHotReloadSessionManager, IProjectHotReloadUpdateApplier
 {
-    private readonly UnconfiguredProject _project;
+    private readonly ConfiguredProject _configuredProject;
+    private readonly UnconfiguredProject _unconfiguredProject;
```

Comment:
Seems unnecessary to store UnconfiguredProject in a field as it can be trivially access via `_configuredProject.UnconfiguredProject`

Reply:
I think it'd be ok to make those changes.